autotest/config.yaml

```diff
@@ -0,0 +1,49 @@
+base_path:
+    base_output_path: /mnt/shared-storage-user/caoweihan/qa-llm-cicd/test_output
+    base_baseline_path: /mnt/shared-storage-user/llmrazor-share/qa-llm-cicd/xtuner_baseline
+
+default_config:
+    train: 
+        resource:
+            gpus_per_task: 8
+            cpus_per_task: 120
+            memory_per_task: 512
+            image: registry.h.pjlab.org.cn/ailab-llmrazor/xtuner:pt28_20251113_22badb0_grouped_router_topk1
+            envs: 
+                - HF_HUB_CACHE=/mnt/shared-storage-user/auto-eval-pipeline/opencompass/models/hf_hub
+    eval:
+        resource:
+            gpus_per_task: 0
+            cpus_per_task: 16
+            memory_per_task: 128
+            image: registry.h.pjlab.org.cn/ailab-puyu/auto-eval:ld_0101_oc_8ee07ac_v3
+            envs: 
+                - HF_HUB_CACHE=/mnt/shared-storage-user/auto-eval-pipeline/opencompass/models/hf_hub
+                - HF_DATASETS_CACHE=/mnt/shared-storage-user/auto-eval-pipeline/opencompass/llmeval/hf_cache
+                - COMPASS_DATA_CACHE=/mnt/shared-storage-user/auto-eval-pipeline/opencompass/llmeval/compass_data_cache
+                - HF_DATASETS_OFFLINE=1
```

Comment:
Could we hide the common path prefix in the runner environment? It would be convenient for different runner (npu、gpu) to use different path